mongodbatlas/resource_mongodbatlas_search_index_test.go

```diff
@@ -21,14 +22,14 @@ func TestAccClusterRSSearchIndex_basic(t *testing.T) {
 		datasourceName        = "data.mongodbatlas_search_indexes.data_index"
 	)
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheckBasic(t) },
+		PreCheck:                 func() { testAccPreCheckSearchIndex(t) },
 		ProtoV6ProviderFactories: testAccProviderV6Factories,
 		CheckDestroy:             testAccCheckMongoDBAtlasSearchIndexDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMongoDBAtlasSearchIndexConfig(projectID, clusterName),
+				Config: testAccSearchIndexConfig(t, projectID, clusterName),
```

Comment:
Can we create the project as part of the test? One problem with this approach is that it may fail if another PR try to run it at the same time (they are using the same project)

Reply:
no, because the beta feature we were talking, we have to use an specific project

Reply:
We could decide to not run it as part of the acceptance test until the feature is released in prod

Reply:
that's an option but to be honest I prefer to leave as it is so we can have more confidence on the vector search changes, later we can change it to create the project in the test.

Reply:
Leaving this thread open to see what other people think about it. Personally, I don't like having tests that will likely be red, this add additional cost as the on-call person, who may not have the context about the feature flag, has to put time to understand why they are failing

Reply:
i don't see why they should be red frequently because of this, all resources have randomized names so it should be fine

Reply:
my understanding is that the test will fail if there are two PRs running it because they are using the same Atlas project. Is my understanding correct?

Reply:
they won't fail because they use random cluster names.

anyway I haven't changed this, current tests are using an existing project.

Reply:
> If we want to change the test to create a project it's a change to the existing tests and I think that can belong to a different PR.

The fact that the tests are not running in GH makes me think that they may fail if run concurrently. We can always remove them if I am correct